app/assets/javascripts/qpixel_api.js

```diff
@@ -12,15 +12,42 @@ window.QPixel = {
     return token;
   },
 
+  /** Counts notifications popped up at any time. */
+  var popped_modals_ct = 0;
+
   /**
    * Create a notification popup - not an inbox notification.
    * @param type the type to apply to the popup - warning, danger, etc.
    * @param message the message to show
    */
   createNotification: function(type, message) {
+    /** Some messages include a date stamp. */
+    var append_date = false;
```

Comment:
Likewise `const` and `let` - just assume I've repeated this for every instance of `var` :)

Reply:
`var` in https://github.com/codidact/landing-page/pull/53/files too?

Reply:
I have yet to review that one properly, but probably :)